File: base16/pygments/pygments_base16/base16-srcery.py
from pygments.style import Style
from pygments.token import (
    Comment, Error, Keyword, Literal, Name, Number, Operator, String, Text
)


class Base16Style(Style):
    base00 = '#1C1B19'
    base01 = '#98BC37'
    base02 = '#FED06E'
    base03 = '#918175'
    base04 = '#68A8E4'
    base05 = '#BAA67F'
    base06 = '#FF5C8F'
    base07 = '#FCE8C3'
    base08 = '#EF2F27'
    base09 = '#FF5F00'
    base0a = '#FBB829'
    base0b = '#519F50'
    base0c = '#0AAEB3'
    base0d = '#2C78BF'
    base0e = '#E02C6D'
    base0f = '#2BE4D0'

    default_style = ''

    background_color = base00
    highlight_color = base02

    styles = {
        Text: base05,
        Error: base08,  # .err

        Comment: base03,  # .c
        Comment.Preproc: base0f,  # .cp
        Comment.PreprocFile: base0b,  # .cpf

        Keyword: base0e,  # .k
        Keyword.Type: base08,  # .kt

        Name.Attribute: base0d,  # .na
        Name.Builtin: base0d,  # .nb
        Name.Builtin.Pseudo: base08,  # .bp
        Name.Class: base0d,  # .nc
        Name.Constant: base09,  # .no
        Name.Decorator: base09,  # .nd
        Name.Function: base0d,  # .nf
        Name.Namespace: base0d,  # .nn
        Name.Tag: base0e,  # .nt
        Name.Variable: base0d,  # .nv
        Name.Variable.Instance: base08,  # .vi

        Number: base09,  # .m

        Operator: base0c,  # .o
        Operator.Word: base0e,  # .ow

        Literal: base0b,  # .l

        String: base0b,  # .s
        String.Interpol: base0f,  # .si
        String.Regex: base0c,  # .sr
        String.Symbol: base09,  # .ss
    }


from string import capwords  # noqa: E402
Base16Style.__name__ = 'Base16{}Style'.format(
    capwords('srcery', '-').replace('-', '')
)
globals()[Base16Style.__name__] = globals()['Base16Style']
del globals()['Base16Style']
del capwords

File: base16/crosh/colors/base16-srcery.js
// Base16 srcery
// Scheme: 

var color_scheme = {
        'base00': '#1C1B19',
        'base01': '#98BC37',
        'base02': '#FED06E',
        'base03': '#918175',
        'base04': '#68A8E4',
        'base05': '#BAA67F',
        'base06': '#FF5C8F',
        'base07': '#FCE8C3',
        'base08': '#EF2F27',
        'base09': '#FF5F00',
        'base0A': '#FBB829',
        'base0B': '#519F50',
        'base0C': '#0AAEB3',
        'base0D': '#2C78BF',
        'base0E': '#E02C6D',
        'base0F': '#2BE4D0',
};

term_.prefs_.set('background-color', color_scheme.base00);
term_.prefs_.set('foreground-color', color_scheme.base05);
term_.prefs_.set('cursor-color', "rgba(186, 166, 127, 0.5)");

term_.prefs_.set('color-palette-overrides', 
                        [color_scheme.base00,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base05,
                        color_scheme.base03,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base07,
                        color_scheme.base09,
                        color_scheme.base0F,
                        color_scheme.base01,
                        color_scheme.base02,
                        color_scheme.base04,
                        color_scheme.base06]);
